### ISSUE TITLE
there is a way to download the plugin?

### ISSUE THREAD

**Issue by arnon001** [GuzioMG/MinecraftFTP#1]

**Comment by GuzioMG** [comment]
Oh god... I completely forgot about this project 😅.

The whole point why I created this plugin was to make it easier for me to work on my server that I run on GCP. The problem with GCP is that it's terribly hard to mange files on there. So, here came MinecraftFTP. I pretty much finished it now (*a long time ago), but I did not publish it anywhere. That's because I had just "that last 10%" of the project to do. No more coding, just cleaning up the repo, configuring `.gitignore`, setting up CIs & GitHub actions, writing `README.MD` and the documentation, publishing to Spigot Plugin Store... You know, the usual stuff people do when their plugin is done and they want to others to use it.

The problem with "that last 10%" is that although it's always a thing in coding (whether it's a game that you want to put on Steam, a library that you want to package onto NPM, Maven, PIP, etc. or a mobile app that you want to put onto App/Play Store), **it's also always the hardest part**. It's the most boring, annoying and usually you find a ton of bugs during that last part of the early development stage in the project's lifecycle. That's why, because I arleady had the project self-compiled onto my server and it worked - I pushed the release date further. Then I was too busy, so I pushed it even further. And further, and further...

Until I forgot about this project completely. **But you just reminded me, thanks to this issue.** And you know what? **I'll do it.** Now that I see that there's some demand for a solution like this, and I see that there's someone waiting... I'll do it. I'll go through the struggle of writing the docs, figure out CI, iron out all the remaining bugs and **I'LL DO IT. This plugin will be published soon!** But since I'm that type of guy who does everything juts before the deadline - I need a clear deadline or else I'll forget once again.

That deadline, to be reasonable, will be set in exactly a month. Right now, in Poland (where I live), we have 15:06 on July the 6th 2021. **On up to August the 6th 2021, at exactly 15:06, expect the plugin to be up and ready to download** (I said up to, because I'll probably get everything done a bit earlier). **I promise.**

_...If you really need it done now and have no time to wait, you can always self-compile it. The code in this repo is complete and working, so as long as you have the technical knowledge and an environment similar enough to mine - you should be able to do that._

**Comment by arnon001** [comment]
If You need any help in write docs, Upload To SpigotMC And Any Think Tell Me And I'll try help,
My discord:Arnon001#2312

**Comment by arnon001** [comment]
i've Found The .jar Of This Repo, And I Can help you find bugs and Errors etc...
but for what version this plugin?

**Comment by GuzioMG** [comment]
First of all, sorry to keep you waiting for an entire day. I was coming back from a trip.

> If You need any help in write docs, Upload To SpigotMC And Any Think Tell Me And I'll try help,
My discord:Arnon001#2312

Ok, thanks! I'll know who to reach out to in case I'll need help. Btw, my Discord ID is Guzio#5010. If I'll need to contact you, expect a message from this user.

> i've Found The .jar Of This Repo, And I Can help you find bugs and Errors etc...
but for what version this plugin?

Ah, yes. That file. I remember why left it there: first of all, that was the only way to actually get the plugin installed on GCP. I would basically run `wget` pointed towards it. Second of all, `.gitignore` is not configured, so binaries find their way into the repo. You can use it because (as I said) the plugin is already functional. It was written for 1.16, but appears to run on 1.17. That's said, I would not recommend it. It's unstable and field-tested only a couple of times. It's most likely very insecure, too - but I'm not an expert in that domain (tho I have enough knowledge to know that allowing players to screw around with the server's file system is probably a bad idea and a rudimentary operator check is probably not enough). If, despite all of that, you still want to test it, I'd appreciate it. But I'm not sure if the feedback would be that helpful to me - I've never been good at collaborative projects and I usually work alone/with a group of close friends. Working on a plugin with someone more or less random would definitely be an interesting experience and I'm looking forward to it.

**Comment by arnon001** [comment]
I Saw That Your Deadline was before 3 days,
do you have any new?

**Comment by GuzioMG** [comment]
Yes. I know. Sorry, it turned out I'm in Hungary (as I mentioned in one of my previous messages, I live in Poland). Basically, my this year's summer vacation are an unplanned mess and up until like... a week ago, I didn't even know I'm going abroad.
Anyway, my point is that unexpectedly, it turned out that I wouldn't be able to keep that deadline because I wouldn't be able to work on the project. I apologize, I should've told you when I found out.
Right now the deadline is getting forcibly pushed until I get back, which is... Whenever that happens, basically. As I said, my summer vacations are a completely unplanned mess.
One thing is for sure, tho. It'll definitely happen before the summer vacation ends. Again, I'm sorry for that happened.

**Comment by arnon001** [comment]
Have Fun!
I guess Its all mess cuz the coronavirus...

**Comment by arnon001** [comment]
You Came back?
cuz a month Passed....

**Comment by GuzioMG** [comment]
Yea, came back a long time ago, actually...
School! School happened! Look at my GitHub status. "School's taking up all of my time." I... Will do it... At some point.

...hopefully
